web/src/styles/wiki.css

```
@@ -2,32 +2,43 @@
-  --wk-amber-bg:        var(--olive-200);
-  --wk-amber-banner:    var(--olive-100);
-  --wk-code-bg:         var(--bg-warm);
-  --wk-display:    var(--font-sans);
+  --wk-amber: var(--olive-500);
+  --wk-amber-text: #B8860B; /* dark amber for readable text on light amber bg */
+  --wk-stale-text: #B94040; /* dark red for readable text on light red bg */
+  --wk-amber-bg: var(--olive-200);
+  --wk-amber-banner: var(--olive-100);
+  --wk-code-bg: var(--bg-warm);
+  --wk-display: var(--font-sans);
   --wk-body-serif: 'Source Serif 4', ui-serif, Georgia, serif;
```

Stylelint (17.9.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)